dsl_parser/framework/parser.py

```diff
@@ -182,10 +275,15 @@ def _traverse_element_type_schema(self, schema, parent_element):
                 return
             for name_holder, value_holder in parent_element.\
                     initial_value_holder.value.items():
+                current_namespace = value_holder.namespace or namespace
+                set_namespace_to_element(current_namespace,
```

Comment:
I think it's more readable to do:
```
if parent_element.add_namespace:
    set_namespace ...
```
Instead of passing `add_namespace` to the function

Reply:
Done

Reply:
Done